src/Lumi/Components/Select.purs

```diff
@@ -289,10 +289,10 @@ select = makeStateless component render
                                     ]
                                 }
                   in
-                    guard
+                    (guard
                       (props.allowMultiple || String.null selectState.searchTerm)
-                      renderedSelectedValues
-                      <> guard props.searchable [ renderNativeInput ]
+                      renderedSelectedValues)
+                      <> (guard props.searchable [ renderNativeInput ])
```

Comment:
This didn't end up changing anything, but I added the parens to make sure the order of operations was correct and it seemed better to leave them in